lib/glific/mails/mail_log.ex

```diff
@@ -15,7 +15,7 @@ defmodule Glific.Mails.MailLog do
   }
 
   @required_fields [:category, :organization_id]
-  @optional_fields [:status, :content, :error]
+  @optional_fields [:status, :content, :error, :inserted_at]
```

Comment:
`inserted_at` and `updated_at` are updated automatically when entry is created and updated so we skip it here.